data/getStatesData.ts

```diff
@@ -107,7 +106,7 @@ const renameLineData = ({
 })
 
 const pushLineToStateDate = (state: StateKeys, date: string) => (
-	line: object,
```

Comment:
Ok so how to type an object? Basically you do `Record<string, any>` or `Record<string, unkown>` if you're hardcore. `{}` is not what you want bc it means `anything that is not null` and `object` is also not what you want because it can be any non-primitive. see https://mariusschulz.com/blog/the-object-type-in-typescript